### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by fedepedron**
Really small PR concerning two things:
* Recovered translation part in SCF.
* Made limits for XC integrals dependent on the precision used.

### CODE REVIEW

**Comment by fedepedron** [comment]
@charlyqchm, @gonzalodm please test.

File: lioamber/faint_cpu/subm_int1.f90 [hunk 1]
```diff
@@ -36,11 +36,11 @@ subroutine int1(En, RMM, Smat, Nuc, a, c, d, r, Iz, ncont, NORM,&
         integer,                       intent(inout) :: natom
 
         double precision, allocatable, intent(in) :: d(:,:)
-        double precision, allocatable, intent(in) :: r(:,:)
+        double precision, intent(in) :: r(natom,3)
```
Comment (ramirezfranciscof) [inline lioamber/faint_cpu/subm_int1.f90]: Are you getting any error because of this? This should not be necessary...

Comment (ramirezfranciscof) [reply lioamber/faint_cpu/subm_int1.f90]: Wowo, I just see this: the problem should not be the assumed shape, but the allocatable descriptor. In fact, unless its being used somewhere below to allocate/deallocate, none of these variables should be described as allocatable...

Comment (fedepedron) [reply lioamber/faint_cpu/subm_int1.f90]: | none of these variables should be described as allocatable
I know, after this I'll make a new PR with quite a lot of changes to int1.

**Comment by charlyqchm** [comment]
It seems that everything is working normally. I tested transport, TDFTB, and open shell and seem to be running normally, with and without CUDA.

**Pull request merged**